thoth/common/openshift.py

```diff
@@ -2325,6 +2325,68 @@ def get_kebechet_template(self) -> Dict[str, Any]:
 
         return self._get_template("template=kebechet")
 
+    def schedule_si_bandit(
+        self,
+        python_package_name: str,
+        python_package_version: str,
+        python_package_index: str = "https://pypi.org/simple",
+        *,
+        job_id: Optional[str] = None,
+    ) -> Optional[str]:
+        """Schedule a bandit security indicator run."""
+        if not self.backend_namespace:
```

Comment:
I think we will want to run it in middletier namespace. The backend namespace is allocated for user requests.